### PR TITLE
fixing issue #888

### PR DESCRIPTION
**Pull request by AdamRussak**
## Description
fixing #888 

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (changes only affecting documentation)

## Checklist

- [x] I have tested my changes locally and ensured they are functioning properly.  Please run the `make build` and `make test` commands.
- [x] I have added/updated unit or e2e tests to cover my changes.
- [ ] I have updated the relevant documentation. If you change commands or arguments, run `make doc-gen` to generate new documentation.

### CODE REVIEW

File: cmd/utils.go [hunk 1]
```diff
@@ -481,3 +502,25 @@ func validateContextTemplate(contextTemplate []string) error {
...
+	if strings.HasPrefix(path, "~/") {
+		path = filepath.Join(homeDir(), path[2:])
+	}
+	// read files info
+	_, err := os.Stat(path)
+	if err != nil {
+		return "", err
+	}
+	return path, nil
+}
```
Comment (sunny0826) [inline cmd/utils.go:526]: Please add unit test to both func

**Review by sunny0826 — CHANGES_REQUESTED** [review]
Excellent, just need to add some unit tests.

**Comment by AdamRussak** [comment]
@sunny0826 done..
i dont see a reason to add e2e, as this is already part of the tests so it is tested.

File: cmd/utils.go (hunk 1 above)
Comment (AdamRussak) [reply cmd/utils.go:526]: added

**Comment by sunny0826** [comment]
> @sunny0826 done.. i dont see a reason to add e2e, as this is already part of the tests so it is tested.

yep, there is no need to add e2e test to this bug fix

**Review by sunny0826 — APPROVED** [review]
LGTM
Thank you for your valuable contribution to this project! Your work is greatly appreciated. 🎆

**Pull request merged**